recipes/fiji-ilastik/meta.yaml

```
@@ -0,0 +1,19 @@
+{% set name = "fiji-ilastik" %}
+{% set version = "1.8.2" %}
+ 
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://maven.scijava.org/service/local/repositories/releases/content/org/ilastik/ilastik4ij/{{ version }}/ilastik4ij-{{ version }}.jar
+  sha256: 7ed05f4941d6d870e70bae3ff7e183a1d750ea5ed63f0efaf6bcbac500c715a0
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - fiji >=20220414
+    - ilastik >=1.4.0b29
```

I'd propose adding some minimal test here. Something along the lines
```suggestion
    - ilastik >=1.4.0b29

test:
  commands:
    - ilastik --help
    - ImageJ --help

```

I mean I know that this wouldn't test whether paths are set up correctly and all, but at least we'd know that all executables are where they are at.

I'm not sure why `ImageJ --help` has a return code of `1`? (This makes the test fail if added like this).